lib/controller/index.js

```diff
@@ -0,0 +1,15 @@
+const program = require('caporal');
+
+const init = require('./init');
+
+module.exports = {
+  start: () => {
```

Comment:
please move the code out of module.exports into a public function which can be exported and called.